env.d.ts

```diff
@@ -1,4 +1,7 @@
 declare module '@env' {
   export const MYANIMELIST_CLIENT_ID: string;
   export const ANILIST_CLIENT_ID: string;
+  export const GIT_HASH: string;
+  export const RELEASE_DATE: string;
+  export const BUILD_TYPE: string;
```

Comment:
```suggestion
  export const BUILD_TYPE: 'Release' | 'Debug';
```
Traitor could forgot, and set BUILD_TYPE = 'release', we would have a weird build name in release app